### ISSUE TITLE
Config and model on Waymo

### ISSUE THREAD

**Issue by alanzhangcs** [PuFanqi23/MonoDGP#9]
Hi, thanks for your excellent work!

When reading your new arxiv paper, I found the experiments results on waymo dataset is quite impressive. 

I'm wondering whether you could release your training and config about waymo dataset. 

Thanks

**Comment by PuFanqi23** [comment]
Thanks for your interest in our work!

While we are currently occupied with other projects and unable to clean up the code, we'd like to share some key implementation insights that might help reproduction:

1、We follow the DEVIANT to convert Waymo to KITTI format. Make sure waymo dataset is complete and eval.py code works properly. The eval code provided by DEVIANT initially gave me all zeros (possibly due to environment issues), so you may need to debug and adapt it.

2、When scaling up batch size with multiple GPUs, remember to increase the learning rate accordingly.

3、Image scaling will affect the geometric perspective relationship, so MonoDGP is highly sensitive to input resolution. You need to change the resolution from (1280, 384) to (768, 512) in dataloader file. 

Hope this helps!

**Comment by alanzhangcs** [comment]
Thanks for your reply.
